app.js

```diff
@@ -0,0 +1,15 @@
+const express = require("express");
+const { handlePSQLErrors } = require("./controller/errror.controller");
+const { getTopics } = require("./controller/news.controller");
+const app = express();
+
+app.use(express.json());
```

Comment:
We don't need this until later tickets - remove for now, then add back in when you actually need it.